### PR TITLE
WIP: Dev

### PR DESCRIPTION
**Pull request by CountCulture**

### CODE REVIEW

File: docs/_schema_tables/AlternateName.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Field Name,Description,Format
-type,"Type: What kind of alternative name is this? Select from 'translation', 'formerName','birth', and 'alias'. List options: [ 'translation', 'former', 'alias', 'birth' ]",string
+type,"Type: What kind of alternative name is this? Select from 'translation', 'formerName','birth', and 'alias'.",string
```
Comment (CountCulture) [inline docs/_schema_tables/AlternateName.csv]: The narrative says it's a list, but the field is a string. Typo?

File: docs/_schema_tables/EntityStatement.csv [hunk 2]
```diff
@@ -1,13 +1,13 @@
 Field Name,Description,Format
 id, See `ID <#id>`__,Object
-type,"Type: What kind of entity is this? The 'registeredEntity' code covers any legal entity created through an act of official registration, usually resulting in an identifier being assigned to the entity. The 'arrangement' code covers artificial entities, described in the data model for the purpose of associating one or more natural or legal persons together in an ownership or control relationship. List options: [ 'registeredEntity', 'arrangement' ]",string
+type,"Type: What kind of entity is this? The 'registeredEntity' code covers any legal entity created through an act of official registration, usually resulting in an identifier being assigned to the entity. The 'legalEntity' code covers other bodies with distinct legal personality (government departments, international institutions etc.).  The 'arrangement' code covers artificial entities, described in the data model for the purpose of associating one or more natural or legal persons together in an ownership or control relationship, but without implying that the parties to this arrangement have any other form of collective legal identity.",string
```
Comment (CountCulture) [inline docs/_schema_tables/EntityStatement.csv]: Should this be a code list?

File: docs/_schema_tables/NullParty.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Field Name,Description,Format
+type,Null type: Why can't a beneficial owner be specified?,string
```
Comment (CountCulture) [inline docs/_schema_tables/NullParty.csv]: Should this be "Why a beneficial owner can't be specified?

File: docs/_schema_tables/AlternateName.csv (hunk 1 above)
Comment (timgdavies) [inline docs/_schema_tables/AlternateName.csv]: I think this is a typo in the script that generates the front-end view. Will get that fixed on master.

**Pull request merged**